packages/contracts/test/InitializeTest.sol

```diff
@@ -202,6 +202,18 @@ contract InitializeTest is BaseTestSetup {
         );
     }
 
```

Comment:
Should we add the tests cases for these reverts too?

```sol
if (_minimumElapsedTime == 0) revert InvalidMinimumElapsedTime();
if (_expirationBufferTime == 0) revert InvalidExpirationBufferTime();
```

Reply:
Addressed in ca2ecb52ced3d2a33e844b2b546c9efb966dc707